js/apps/system/_admin/aardvark/APP/frontend/js/routers/startApp.js

```diff
@@ -13,6 +13,27 @@
       }
     });
 
+    const updateOptions = function(options) {
+      const update = { ...options };
+      update.headers = {
+        ...update.headers,
+        'X-Arango-Frontend': 'true'
+      };
+      var currentJwt = window.arangoHelper.getCurrentJwt();
+      if (currentJwt) {
+        update.headers = {
+          ...update.headers,
+          Authorization: 'bearer ' + currentJwt
+        };
```

Comment:
Same here. While this might be simpler now, it will be make the code difficult to debug later. We should make shallow copies and avoid mutation in general